src/main/java/omero/cmd/graphs/DuplicateI.java

```diff
@@ -1086,6 +1123,12 @@ private void abort() {
     @Override
     public void finish() {
         managedRepository.clearFileCreationListener();
+        if (diskUsage.isEmpty()) {
+            LOGGER.info("no file bytes copied");
```

Comment:
Nice.  This would be nice in the response, but I assume that's sufficiently troublesome water that it should be postponed?

Reply:
At least for this PR I'm trying to avoid adjusting the API. I also figure we might need a round of UX thinking to be happy we're breaking it down in the desired way before exposing it in that form.